src/main/java/com/gmail/val59000mc/listeners/PlayerDeathListener.java

```diff
@@ -37,6 +40,7 @@ public void onPlayerDeath(PlayerDeathEvent event){
 		MainConfiguration cfg = gm.getConfiguration();
 		UhcPlayer uhcPlayer = pm.getUhcPlayer(player);
 
+
```

Comment:
This return got in here somehow, it's not a big thing of course but it doesn't need to be there either.